.circleci/config.yml

```diff
@@ -373,10 +373,6 @@ commands:
             # Use helm to template our chart against all kube versions
             TEMPLATE_DIR=$(mktemp -d)
             for kube_version in ${CURRENT_KUBE_VERSIONS}; do
-              # Skip 1.28.0 until supported by kubeconform
```

Comment:
I think it's not related, it looks like it's related to kube version isn't it ?

Reply:
Good eye.  But yeah, **not** related.  It's the Kubernetes version which also released this last week. 😉